pyscf/df/df_jk.py

```diff
@@ -304,7 +304,7 @@ def get_jk(dfobj, dm, hermi=1, with_j=True, with_k=True, direct_scf_tol=1e-13):
                          ctypes.c_int(naux), ctypes.c_int(nao),
                          (ctypes.c_int*4)(0, nocc, 0, nao),
                          null, ctypes.c_int(0))
-                    vk[k] += lib.dot(buf1.T, buf1)
+                    vk[k] += numpy.matmul(buf1.T, buf1)
```

Comment:
Please use `lib.dot` . due to threading parallel efficiency issue in openblas and mkl, the dsyrk in np.dot is often slower than lib.dot for larger number of threads. If you wish to optimize this line using dsyrk, please implement the function in C with explicit openmp parallelization as the lib.dot does.

Reply:
Good point about dsyrk. I did not think of that and will fix this.